test/e2e/catalog_source_OS4.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: operators.coreos.com/v1alpha1
```

Comment:
@sbose78 @corinnekrych @baijum @tinakurian can we please always add **at least** one comment line that describes what we're looking at? I'm a bit overwhelmed by the amount of YAML everybody happily pushes. This is an undocumented piece of config file that nobody knows how to read or maintain in half a year. Please be careful! I'm also happy with URLs to specific fields in the K8s domain.

Reply:
I have added a link to OLM.